modules/hal_nordic/nrfx/nrfx_config_nrf54l15_enga_application.h

```diff
@@ -1517,15 +1517,6 @@
 #define NRFX_UARTE_CONFIG_TX_LINK 1
 #endif
 
-/**
- * @brief NRFX_UARTE_CONFIG_RX_CACHE_ENABLED
```

Comment:
This should be also removed: https://github.com/zephyrproject-rtos/zephyr/blob/ef4747c939231bab5603c66cec5054219dbc6513/modules/hal_nordic/nrfx/nrfx_config.h#L815-L817
and this: https://github.com/zephyrproject-rtos/zephyr/blob/ef4747c939231bab5603c66cec5054219dbc6513/modules/hal_nordic/nrfx/Kconfig#L951-L957

Reply:
IMO it shouldn't be removed, because nrf54h_* devices still make use of it - it was only removed for nrf54l_* devices. Please take a look at diff for nrfx_config template for nrf54h app, for example: https://github.com/zephyrproject-rtos/hal_nordic/commit/884c4d61746bc35fbd379c169fc87ddb56c6461d#diff-02800d39e6b44178911a9be5fda6ec7bdf90fd37e76a1f8687ab39f433653659

Reply:
Ok, I somehow thought that this feature was removed entirely from the driver. Now I see that I misunderstood.